docs/source/topics/proc_setting-up-remote-server.adoc

```diff
@@ -38,14 +38,14 @@ $ sudo dnf install haproxy /usr/sbin/semanage
 . Modify the firewall to allow communication with the cluster:
 +
 ----
-$ sudo systemctl start firewalld
-$ sudo firewall-cmd --add-port=80/tcp --permanent
-$ sudo firewall-cmd --add-port=6443/tcp --permanent
-$ sudo firewall-cmd --add-port=443/tcp --permanent
-$ sudo systemctl restart firewalld
+$ sudo systemctl enable --now firewalld
+$ sudo firewall-cmd --add-service=http --permanent
+$ sudo firewall-cmd --add-service=https --permanent
+$ sudo firewall-cmd --add-service=kube-apiserver --permanent
+$ sudo firewall-cmd --reload
 ----
 
-. For SELinux, allow listening to TCP port 6443:
+. For SELinux, allow listening to TCP port 6443 (kube-apiserver):
```

Comment:
I think selinux will only allow haproxy to use a very specific set of ports, most likely just 443 and 80. This semanage command tells selinux that it's fine for haproxy to also use port 6443. And we need to expose 6443 because this is the port used by the openshift/kube apiserver, ie the entrypoint used by `oc` and all tools interacting with the cluster.

Reply:
I've updated the PR to include this information.

Reply:
Thank you! The changes look good to me. :+1: